frontend/src/components/print/config/ProgramConfig.vue

```diff
@@ -37,5 +43,8 @@ export default {
       dayOverview: true,
     }
   },
+  design: {
+    multiple: true,
```

Comment:
minor:
polymorphism is cool, but i don't know if 5 times a false makes it worth.
I don't have a better idea right  now (except for a "default implementation" with false.